src/tests/imgui_demo.edn

```diff
@@ -438,7 +438,7 @@
 (defnode main)
 (defloop ui-chain
   (GFX.MainWindow
-   :Title "UI Chainblocks Example" :Width 1280 :Height 768 :Debug true :Fullscreen false
```

Comment:
what happened to fullscreen btw?

Reply:
Yeah it got removed during the rewrite.
Will include in a later PR but made a note so I don't forget (https://github.com/orgs/fragcolor-xyz/projects/1#card-79476112)